pyuvdata/uvfits.py

```diff
@@ -23,6 +23,30 @@ class UVFITS(UVData):
     uvfits_required_extra = ['antenna_positions', 'gst0', 'rdate',
                              'earth_omega', 'dut1', 'timesys']
 
+    def _fixhistory(self):
+        """
+        each line of a uvfits history has a maximum of 72 characters
+        which is not a limitation for other formats (miriad, ms, etc...)
+        this function goes through history and merges any lines with 72 chars
+        with the line that follows it. If there is actually a history line that
```

Comment:
What do you mean "actually a history line that should be 72 chars"  Do you mean "must be 72 chars"?  like ascii art?